recipes/elegansbot/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "elegansbot" %}
+{% set version = "1.0.2" %}
+{% set python_min = "3.8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source: 
+  url: https://files.pythonhosted.org/packages/b1/07/3a1453d6e2df7f59e7a5d8fe3a652afbf7d77f23479bddd6767a18c1b0b1/elegansbot-1.0.2.tar.gz
```

Comment:
Please use the canonical url found here:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16